mocks/mocks.py

```diff
@@ -21,6 +21,23 @@
 from gunagala.utils import ensure_unit
 
 from mocks.utils import load_yaml_config
+from mocks import utils
+
+
+def prepare_data(particle_pos,
```

Comment:
This function prepares the data and it is possible that the data which Chris will provide for us does not need to be prepared.

Reply:
maybe rename to 
```suggestion
def crop_simulation_data(particle_pos,
```